packages/ubuntu_wsl_setup/lib/main.dart

```diff
@@ -17,23 +18,31 @@ screens, yet allowing user to overwrite any of those during setup.
   ''',
     );
   })!;
-  var variant = Variant.WSL_SETUP;
+  final variant = ValueNotifier<Variant?>(null);
   List<String>? serverArgs;
-  if (options['reconfigure'] == true) {
-    variant = Variant.WSL_CONFIGURATION;
-  }
   if (options['prefill'] != null) {
     serverArgs = ['--prefill', options['prefill']];
   }
   runWizardApp(
-    UbuntuWslSetupApp(
-      initialRoute: options['initial-route'],
-      reconfigure: options['reconfigure'] == true,
+    ValueListenableBuilder<Variant?>(
+      valueListenable: variant,
+      builder: (context, value, child) {
+        return UbuntuWslSetupApp(
+          variant: value,
+          initialRoute: options['initial-route'],
```

Comment:
The route name shouldn’t match the mode? (not a blocker for this PR, but just a thought)

Reply:
The --initial-route argument is not used for selecting the mode. The wizard UI is built based on the above variant value. The --initial-route argument allows jumping to a specific page regardless of the mode. It's mostly useful for integration testing (to avoid repeatedly navigating through all pages in every test case).

Reply:
For example, this would jump straight to the profile setup:
```
ubuntu-wsl-setup --initial-route="/profile-setup"
```
It's a hidden argument and not useful to users since they don't know the route names that are an internal implementation detail. However, in integration tests, it's useful when there are several different page-specific test cases. For example, if we had multiple test cases specifically for the profile setup, this argument would make it possible to avoid navigating through the language selection page in every profile setup-specific test case.

Reply:
ah, ack, it’s not the name of the route itself but the entry point of the current route, got it! :)